src/utils/formHelperFunctions.js

```diff
@@ -311,13 +314,22 @@ function setFormElementError({
       elements.forEach((e) => e.setCustomValidity(message));
     }
     if (triggerErrors) formRef.current.reportValidity();
-    return !formRef.current.checkValidity();
+    invalid = !formRef.current.checkValidity();
   } else if (errorType === 'inline') {
     if (fieldKey) inlineErrors[fieldKey] = { message };
     if (triggerErrors)
       setInlineErrors(JSON.parse(JSON.stringify(inlineErrors)));
-    return Object.values(inlineErrors).find((data) => data.message);
+    invalid = Object.values(inlineErrors).find((data) => data.message);
```

Comment:
is this going to be setting `invalid` to something other than a boolean? seems odd if so, but I also could be missing how the shape of `inlineErrors` actually goes

Reply:
now coercing to a boolean